### ISSUE TITLE
Completely remove Ubuntu touch specific parts 

### ISSUE THREAD

**Issue by mariogrip** [Halium/projectmanagement#1]

**Comment by Flohack74** [comment]
Also this belongs probably to this issue:
`acp: file '/home/florian/halium-5.1/halium/prebuilt-initrd/initrd.img-touch' does not exist`
when building the halium-5.1 ATM...

**Comment by z3ntu** [comment]
@Flohack74 Can you please test https://github.com/Halium/android_build/pull/5?

**Comment by Flohack74** [comment]
Yes can be that this works, but I got another error, so until I fix this one I wont confirm that I get a clean image :)
